spec/support/issue.cr

```diff
@@ -0,0 +1,23 @@
+class Issue < BaseModel
+  COLUMN_SQL = "issues.id, issues.status, issues.role"
+
+  avram_enum Status do
+    Opened = 0
+    Closed = 1
+    Duplicated = 2
+  end
+
+  avram_enum Role do
+    Issue = 0
+    Bug = 1
+    Critical = 2
+  end
+
+  table do
+    column status : Issue::AvramStatus
```

Comment:
Does this have to be namespaced? Or can you just do `column status : AvramStatus`?

Reply:
Sadly yes, it is generating a class directly under Issue. So it has to be. Haven't found it possible. I might be wrong and will be happily wrong here.

Reply:
`column status : AvramStatus` (without the leading `Issue::`) should work OK.

I'm using a modified version of this PR, as suggested in my reviews, and in my case, `column status : Status` (without the `Issue::` namespace or `Avram` prefix) works OK.